src/fixtures/es_archives/visualize/mappings.json

```diff
@@ -0,0 +1,110 @@
+{
+  "type": "index",
+  "value": {
+    "index": ".kibana",
+    "settings": {
+      "index": {
+        "number_of_shards": "5",
+        "number_of_replicas": "1"
+      }
+    },
+    "mappings": {
+      "index-pattern": {
+        "properties": {
+          "fields": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
```

Comment:
Since only master right now is using keywords, what's the gameplan here for getting this into prior versions?  Backport with an additional commit to convert these to strings?